tools/profileBuilder/Gopkg.toml

```diff
@@ -1,7 +0,0 @@
-[[constraint]]
```

Comment:
no need anymore ?

Reply:
No, it was a subset of the root Gopkg.toml.